backend/src/dependencies/store.py

```diff
@@ -65,12 +66,12 @@ def get_deps_to_install(dependencies: List[DependencyInfo]):
 
 def install_dependencies_sync(
     dependencies: List[DependencyInfo],
-):
+) -> int:
     dependencies_to_install = get_deps_to_install(dependencies)
     if len(dependencies_to_install) == 0:
-        return
+        return -1
```

Comment:
Why would "there are no deps that need to be installed" be an error?

Reply:
Sorry i should have taken this out when i was done debugging it